src/components/BuyModal.re

```diff
@@ -78,17 +78,17 @@ module Transaction = {
     let getMax = [%bs.raw {| (first, second) => Math.max(first,second) |}];
     let currentPriceFloatWithMinimum = getMax(. currentPriceFloat, 0.005);
     let defaultPriceValue =
-      Js.Float.toFixedWithPrecision(
+      toFixedWithPrecisionNoTrailingZeros(
```

Comment:
This helper function I made doesn't add unnecessary trailing zeros.

ie `0.10000001` to precision of 3 becomes "0.1" not "0.100"